tock/utilization/unit.py

```diff
@@ -39,22 +39,22 @@ def _get_unit_billing_data(unit, recent_periods=None, fiscal_year=False):
     else:
         start, end, utilization = utilization_report(users)
 
-    data = utilization.values('username', 'billable', 'total')
+    data = utilization.values('username', 'billable', 'target')
 
     # Build context for each employee
     output_data = [{
             'username': employee['username'],
-            'total': employee['total'],
+            'denominator': employee['target'],
```

Comment:
super minor, but I wonder if in this context it would be better to call it `target` rather than denominator. Your call, though.

Reply:
🤷  I started with `target` but then imagined a world next week where we wanted to render `total` again 😄